.jscsrc

```diff
@@ -0,0 +1,45 @@
+{
+    "requireOperatorBeforeLineBreak": true,
+    "maximumLineLength": {
+      "value": 100,
+      "allowComments": true,
+      "allowRegex": true
+    },
+    // disabled due to https://github.com/jscs-dev/node-jscs/issues/915
+    // "validateIndentation": 2,
```

Comment:
I'd like to land this as is, and remove later once I address the remaining issues.